### PR TITLE
(azcopy10) Migrate package to new repository

### PR DESCRIPTION
**Pull request by corbob**
## Description

This pull request adds the `azcopy10` package which pauby is currently a maintainer for.

## Motivation and Context

As part of the effort to maintain [critical Chocolatey packages](https://github.com/chocolatey-community/chocolatey-packages/issues/1904), this package is being moved to the Chocolatey Community Chocolatey Packages GitHub Repository.

## How Has this Been Tested?

1. Normal installation and uninstallation through the chocolatey-test-environment
2. running update.ps1 directly in a Windows 10 VM.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/jbpaux/chocolatey-packages)
- [ ] https://github.com/jbpaux/chocolatey-packages/pull/3
- [ ] The [migration guidelines](https://github.com/chocolatey-community/chocolatey-packages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/azcopy10/azcopy10.nuspec
Comment (jbpaux) [inline automatic/azcopy10/azcopy10.nuspec:11]: You should update this line to target path

File: automatic/azcopy10/azcopy10.nuspec
Comment (jbpaux) [inline automatic/azcopy10/azcopy10.nuspec:12]: This URL too

**Review by jbpaux — CHANGES_REQUESTED** [review]

**Comment by jbpaux** [comment]
More other, as the (very) old azcopy package is I think barely used anymore, shouldn't we considerate :
- Replace azcopy package with content of azcopy10 package (and maybe create an azcopy8 package with old content if people still want to use an old version)
- Publish azcopy 10.X as azcopy package
- "Deprecate" azcopy10 package by putting it as a dependency of azcopy package so people can migrate to it

What do you all think ? @pauby ?

**Comment by pauby** [comment]
@jbpaux I think we should discuss that over on https://github.com/chocolatey-community/chocolatey-packages/issues/1904 as it doesn't affect the PR.

File: automatic/azcopy10/azcopy10.nuspec
Comment (corbob) [reply automatic/azcopy10/azcopy10.nuspec]: I've updated this.

File: automatic/azcopy10/azcopy10.nuspec
Comment (corbob) [reply automatic/azcopy10/azcopy10.nuspec]: I've updated this, but I'm not 100% sure on how it'll work. I've updated it to a commit that's part of this PR, but if the commits change at all before merge then I don't think it'll be valid.

**Comment by jbpaux** [comment]
Done @corbob (you miss the second m in `chocolatey-community` I think but I fixed it). It's pending approval.

**Review by pauby — APPROVED** [review]

File: automatic/azcopy10/azcopy10.nuspec
Comment (pauby) [reply automatic/azcopy10/azcopy10.nuspec]: `iconUrl` has been updated to `https://cdn.jsdelivr.net/gh/chocolatey-community/chocolatey-packages@de115ff0e16d124c2099b8f95bd08bb92b6e4918/icons/azcopy10.png`. I'm going to resolve this conversation.

**Comment by pauby** [comment]
Tested this and update, install and uninstall work fine. I've added `chocolatey-community` user to the package, removed @jbpaux, and I've left myself there until the first package is published, _just in case_. After that I'm happy to be removed.

@jbpaux Thank you very much for all your hard work with this package. And thank you for your patience while I tried to find the time to migrate.

Thank you @corbob for migrating this package over.

**Pull request merged**
